assets/react/components/GridItem.jsx

```diff
@@ -1,13 +1,13 @@
 import React from "react";
 import styled from 'styled-components'
-import Icon from './Icons/Icons';
+import Icon from './icon';
 
-const GridItem = ({ variant, description, image, exposed, tileIcons, tileBorders }) => {
+const GridItem = ({ variant, description, image, exposed, showIcons, showBorders }) => {
```

Comment:
Should we not keep the naming `tileIcons` etc. to better differentiate tile and expose?

Reply:
done